src/OmnichannelChatSDK.ts

```diff
@@ -1501,6 +1506,156 @@ class OmnichannelChatSDK {
+                    return Promise.reject("surveyInviteLinkResponse is null.");
+                }
+            } else {
+                return Promise.reject("Post Chat is not enabled from admin side.");
+            }
+        } catch (ex) {
+            this.scenarioMarker.failScenario(TelemetryEvent.GetPostChatSurveyContext, {
+                ConversationId: conversationId,
+                RequestId: this.requestId,
+                ExceptionDetails: JSON.stringify(ex)
+            });
+
+            return Promise.reject("Retrieving post chat context failed " + JSON.stringify(ex));
+        }
+    }
+
+    public async initializePostChatRenderer(): Promise<void> {
+        this.scenarioMarker.startScenario(TelemetryEvent.InitializePostChatRenderer, {
+            RequestId: this.requestId
+        });
+
+        try {
+            let msfpEmbedScript = getMsfpEmbedScript();
+            let elm = document.createElement('div');
+            elm.innerHTML = msfpEmbedScript;
+            let scriptNodes = elm.getElementsByTagName('script');
+            let styleNodes = elm.getElementsByTagName('link');
+            for (let i = 0; i < scriptNodes.length; i++) {
+                let node = scriptNodes[i];
+                if (node.src) {
+                    let tmpNode = document.createElement('script');
+                    tmpNode.type = node.type;
+                    tmpNode.src = node.src;
+                    tmpNode.async = false;
+                    if (!window.document.head.contains(tmpNode)) {
+                        document.getElementsByTagName('head')[0].appendChild(tmpNode);
+                    }
+                }
+                if (node.innerHTML) {
+                    let tmpNode = document.createElement('script');
+                    tmpNode.innerHTML = node.innerHTML;
+                    tmpNode.async = false;
+                    if (!window.document.head.contains(tmpNode)) {
+                        window.document.head.appendChild(tmpNode);
+                    }
+                }
+            }
+            for (let i = 0; i < styleNodes.length; i++) {
+                let node = styleNodes[i];
+                if (node.href) {
+                    let tmpNode = document.createElement('link');
+                    tmpNode.type = node.type;
+                    tmpNode.rel = node.rel;
+                    tmpNode.href = node.href;
+                    if (!window.document.head.contains(tmpNode)) {
+                        document.getElementsByTagName('head')[0].appendChild(tmpNode);
+                    }
+                }
+            }
+        } catch (ex) {
+            this.scenarioMarker.failScenario(TelemetryEvent.InitializePostChatRenderer, {
+                RequestId: this.requestId,
+                ExceptionDetails: JSON.stringify(ex)
+            });
+        }
+    }
+
+    public async renderPostChatSurvey(containerId: string, postChatContext: PostChatContext): Promise<void> {
```

Comment:
Can we add a check of the platform & prevent this from running on other platforms than Web?